src/components/Forms/Card/Card.stories.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from "react";
+import { Meta, Story } from "@storybook/react";
+import { Card } from "./index";
+import EditCard from "./EditCard";
```

Comment:
instead of kayak gini, maybe export semuanya dari `index.ts` (EditCard, Card), terus import dua-duanya sekaligus :thinking: ?

Reply:
changed
<img width="339" alt="image" src="https://user-images.githubusercontent.com/98725806/178643750-0022c036-2176-4bc2-8da4-3a94db1c1de6.png">